src/main/resources/swagger/api-docs.yaml

```diff
@@ -264,6 +267,8 @@ paths:
       responses:
         204:
           description: Successfully created resource
+          schema:
+            $ref: '#/definitions/Resource'
```

Comment:
This api does not actually return anything. What was here before was correct.

Reply:
this means that the addition of Resource, ResourceId and ResourceTypeName are not required.